### ISSUE TITLE
nodeList still in can-stache?

### ISSUE THREAD

**Issue by justinbmeyer** [canjs/can-stache#719]
https://github.com/canjs/can-stache/blob/1e9dd197d3ad21999cf217532cac9853a3178f90/expressions/call.js#L99

```js
			if(func[isViewSymbol] === true) {
				// if not a scope, we should create a scope that
				// includes the template scope
				if(!(args[0] instanceof Scope)){
					args[0] = scope.getTemplateContext().add(args[0]);
				}
				// and include nodeLists
				args.push(helperOptions.nodeList);
			}
```

We don't need to pass that second argument (nodeList). I think that should be removed.

**Comment by justinbmeyer** [comment]
This part:

```js
				if(!(args[0] instanceof Scope)){
					args[0] = scope.getTemplateContext().add(args[0]);
				}
```

Also makes it so `{{ foo(val='bar') }}` always creates a new context.

**Comment by matthewp** [comment]
I'm updating the docs so I'll remove this reference.

**Comment by matthewp** [comment]
Closed by #720
